lib/loopback-proxy-app/legacy-loader.component.ts

```diff
@@ -35,6 +36,8 @@ export class LegacyLoaderComponent implements Component {
     this.authTenant = _.get(config, 'services.auth.tenant') || _.get(config, 'services.auth.organization') || 'ls';
     this.authUrl = _.get(config, 'facility.shell.Auth.Url') || _.get(config, 'auth.url') || 'https://a.labshare.org/_api';
     this.authAudience = _.get(config, 'services.auth.audience') || 'ls-api';
+    this.mountPath = _.get(config, 'services.mountPath') || '/:facilityId';
```

Comment:
facilityId should not be defined at services level even as a default value

Reply:
should we start using Typescript 3.7.0 conditional chaining instead of `.get`:
`config?.services?.mountPath?`

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html

Reply:
@dkrantsberg